src/picongpu/include/simulation_defines/param/synchrotronPhotons.param

```diff
@@ -36,6 +40,9 @@ BOOST_CONSTEXPR_OR_CONST float_64 SYNC_FUNCS_BESSEL_INTEGRAL_STEPWIDTH = 1.0e-3;
 /** Number of sampling points of the lookup table */
 BOOST_CONSTEXPR_OR_CONST uint32_t SYNC_FUNCS_NUM_SAMPLES = 8192;
 
-} // namespace bremsstrahlung
+/** Photons softer than this ratio: (2pi / omega) / DELTA_T, are neglected */
+BOOST_CONSTEXPR_OR_CONST float_64 SOFT_PHOTONS_CUTOFF_RATIO = 1.0;
```

Comment:
This cutoff is to avoid bad numerical rounding and double counting. It has nothing to do with the event generator.